app/utils/report/templates/test.txt

```diff
@@ -9,6 +9,9 @@ Test results summary
   Kernel:  {{ kernel }}
   URL:     {{ git_url }}
   Commit:  {{ git_commit }}
+{%- if image_type != 'kernel-ci' %}
```

Comment:
I think using `kernel-ci` as a special value here is a bit fragile, if this ever changes in the base template then it'll break.  But on the other hand, showing this field in every test email report would probably be more confusing than helpful.